src/e2e/step-definitions/common.ts

```diff
@@ -51,6 +52,16 @@ Given(
   },
 );
 
+Given(
+  "the page is done searching and reloaded",
+  { timeout: ONE_MINUTE_IN_MS },
+  async function (this: CustomWorld) {
+    await this.page.waitForTimeout(THREE_SECONDS_IN_MS);
+    await this.page.reload();
+    await this.page.waitForLoadState("networkidle", { timeout: 5000 });
```

Comment:
Let's introduce a named `FIVE_SECONDS_IN_MS` const for this one

Reply:
Networkidle was not really recomended iirc. There might be background jobs/workers that would make these 5 seconds expire even though the page is loaded